index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment:
The tags h2 can not has the attribute hidden. Maybe you need a input with this property

Reply:
<input hidden> Game over, refresh to play again. 😭 <input>